cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.3.7",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
```

Comment:
2.2.5 is the version used in this file before we bumped to etcd 3.0.3: https://github.com/kubernetes/kubernetes/commit/456c43c22d91511b0a489faee545fad38fe67f11#diff-675fb66c5e2b28dcd560d0b3a2f8e0fcL13